geoportal/geoportailv3_geoportal/static-ngeo/js/Mymaps.js

```diff
@@ -485,6 +485,9 @@ exports.prototype.setMapId = function(mapId) {
   this.stateManager_.updateState({
     'map_id': this.mapId_
   });
+
+  // v4 Set map id MyMaps in lib
+  useAppStore().setMapId(this.mapId_);
```

Comment:
If I get it right this is not used anymore for the message of this PR, but kept for later, right?

Reply:
yes indeed